extensions/ramalama/src/api/connection-event.ts

```diff
@@ -0,0 +1,28 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+// use zod for the json event received by docker/podman
+
+import z from 'zod';
+
+export const DockerEventSchema = z.object({
+  status: z.string().optional(),
```

Comment:
In https://docs.docker.com/reference/cli/docker/system/events/#format-the-output, it seems that both Status and Type start with an uppercase?

Reply:
suggestion: status seems unused, maybe remove it?

Reply:
if you look at your link but at the bottom (where it says what we have using json) you'll see that status has no uppercase